scripts/breakpad-taskcluster.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# TaskCluster build script for building Breakpad.
+# Submit a task like this to build:
+: <<'EOF'
+{
+	"created": "2016-04-09T00:49:49.410Z",
+	"deadline": "2016-04-09T01:49:49.410Z",
```

Comment:
should this be dynamically set? since this deadline is now in the past, I expect future jobs to insta-fail.